<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseBendParameters</class>
 <widget class="QWidget" name="BaseBendParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BaseBend Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="labelBaseStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Base object is a {closed/open} profile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="errLabel">
     <property name="styleSheet">
      <string notr="true">color: Red</string>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="extrusionParameters">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Extrusion Parameters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Material Thickness</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Gui::QuantitySpinBox" name="materialThickness">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Thickness of sheet material used by this object</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="unit" stdset="0">
          <string>mm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="symRevCheckboxes">
       <item>
        <widget class="QCheckBox" name="checkMidPlane">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Applies length symmetrically to sketch plane</string>
         </property>
         <property name="text">
          <string>Symmetric to plane</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkReversed">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Flips the direction of extrusion from the sketch plane&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Reversed</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="bendParameters" native="true">
     <layout class="QVBoxLayout" name="bendParameters_layout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Bend Parameters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Flange Length:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Gui::QuantitySpinBox" name="flangeLength">
          <property name="toolTip">
           <string>Length to extrude the profile</string>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="unit" stdset="0">
           <string>mm</string>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Bend Side:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="bendSide">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Where to create material relative to the sketch profile&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="currentText">
           <string>Outside</string>
          </property>
          <item>
           <property name="text">
            <string>Outside</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inside</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Middle</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Bend Radius:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Gui::QuantitySpinBox" name="bendRadius">
          <property name="toolTip">
           <string>Internal radius of bend corners</string>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="unit" stdset="0">
           <string>mm</string>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkUpdateView">
     <property name="toolTip">
      <string>Show a live preview of the edited object</string>
     </property>
     <property name="text">
      <string>Update view</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
